RELEASING.md

```diff
@@ -242,22 +242,44 @@ At some point in a java version's lifecycle, the JDK version will be maintained
 - [jdk-updates-dev](https://mail.openjdk.java.net/mailman/listinfo/jdk-updates-dev)
 When this occurs, usually a TSC member will create the `jdk<version>u` update repo ([example of the JDK11u one](https://github.com/AdoptOpenJDK/openjdk-jdk11u)) via our Skara mirroring jobs that pull in the commit and tag info from the Mercurial repository. To find out more about Skara and our other mirroring jobs, see https://github.com/AdoptOpenJDK/openjdk-mirror-scripts.
 
+*New Adopt mirror repo creation, by an AdoptOpenJDK github Admin:*
+
+1. Create a new empty repo AdoptOpenJDK/openjdk-jdkNNu
+2. Rename mirror job from https://ci.adoptopenjdk.net/view/git-mirrors/job/git-mirrors/job/git-skara-jdkNN to https://ci.adoptopenjdk.net/view/git-mirrors/job/git-mirrors/job/git-skara-jdkNNu
+3. Update mirror job "Execute shell" to pass jdkNNu as parameter to bash ./skaraMirror.sh jdkNNu
+4. Run the renamed job twice, first one will fail due to empty repo, 2nd run should succeed.
+5. Add the AdoptOpenJDK.md "marker" text file to both branches "dev" and "release".
+
 When the repo has been created, a few changes to the codebase will be necessary where the code references a jdk version but not it's new update version. I.e. `jdk11` became `jdk11u` when it was moved to an update repository.
 
 *If a product is to be moved to an update repo, follow these steps in chronological order to ensure our builds continue to function:*
 
-1. Update the [configurations](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/jobs/configurations)
+1. ci-jenkins-pipelines: Update the [configurations](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/jobs/configurations)
 Rename the nightly build targets file (it will be named `jdkxx.groovy`, [example here](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk15u.groovy)) to be `jdkxxu.groovy`. Do the same for the pipeline config file (named `jdkxx_pipeline_config.groovy`, [example here](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk15u_pipeline_config.groovy)).
 
-2. Update the `javaToBuild` from `jdkxx` to `jdkxxu` inside the [pipeline job](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/tree/master/pipelines/build) that is being shifted to an update repository.
+2. ci-jenkins-pipelines: Update version from `jdkxx` to `jdkxxu` inside [docs/generateBuildMatrix.sh](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/docs/generateBuildMatrix.sh)
+
+3. ci-jenkins-pipelines: Run on "linux" platform your updated docs/generateBuildMatrix.sh script to generate the updated README.md table to be updated at the end of [README.md](https://github.com/AdoptOpenJDK/ci-jenkins-pipelines/blob/master/README.md#build-status)
+
+4. ci-jenkins-pipelines: PR "run tests".
```

Comment:
I know what this means, and you know what this means, but I think it should be clearer.

How about: "In the Pull Request, post a comment saying "run tests". This should execute tests a, b, and c against your changes. Monitor their progress via the links automatically added to the PR, and ensure they all pass.

Reply:
I've removed this bullet as it's not needed, the changes should be dealt with like any other github change...